handlers/photo-audio-haptics-handler/README.md

```diff
@@ -0,0 +1,16 @@
+# Photo Audio Haptics Handler
+
+![license: AGPL](https://camo.githubusercontent.com/b53b1136762ea55ee6a2d641c9f8283b8335a79b3cb95cbab5a988e678e269b8/68747470733a2f2f696d672e736869656c64732e696f2f62616467652f6c6963656e73652d4147504c2d73756363657373) [GitHub Container Registry Package](https://github.com/Shared-Reality-Lab/IMAGE-server/pkgs/container/image-handler-photo-audio)
```

Comment:
Wrong package link

Reply:
Whoops, my bad. Fixed the link and added a short description on how the location information is intended to be used by haptics.